Tensile/TensileRetuneLibrary.py

```diff
@@ -50,13 +50,16 @@ def parseCurrentLibrary(libPath, skipRK):
 
     # process exactLogic into ProblemSizes
     sizes = []
-    for (size, mapping) in exactLogic:
-        if skipRK:
-            sol = solutions[mapping[0]]
-            if sol["ReplacementKernel"]:
-                continue
+    if sizePath is None:
+        for (size, mapping) in exactLogic:
+            if skipRK:
+                sol = solutions[mapping[0]]
+                if sol["ReplacementKernel"]:
+                    continue
 
```

Comment:
This isn't related to your change, but should skipRK also be skipping custom kernels? The isCustomKernelConfig function could be used to test for that as well.

Reply:
The option only exists because I needed it for a specific use case, but if we need to filter out custom kernels in the future for whatever reason this is good to know about.